shared/src/business/entities/DocketEntry.isUnservable.test.ts

```diff
@@ -0,0 +1,32 @@
+import { DocketEntry } from './DocketEntry';
+import { MINUTE_ENTRIES_MAP, UNSERVABLE_EVENT_CODES } from './EntityConstants';
+import { applicationContext } from '../test/createTestApplicationContext';
+
+describe('isUnservable', () => {
+  it('should return true when the docketEntry eventCode is in the list of minute entries', () => {
+    const minuteEntry = new DocketEntry(
+      { eventCode: MINUTE_ENTRIES_MAP.captionOfCaseIsAmended.eventCode },
+      { applicationContext },
+    );
+
+    expect(DocketEntry.isUnservable(minuteEntry)).toBe(true);
+  });
+
+  it('should return true when the docketEntry event code is in the list of unservable event codes', () => {
+    const unservableDocketEntry = new DocketEntry(
+      { eventCode: UNSERVABLE_EVENT_CODES[0] },
+      { applicationContext },
+    );
+
+    expect(DocketEntry.isUnservable(unservableDocketEntry)).toBe(true);
+  });
```

Comment:
Perhaps this could include every item in the `UNSERVABLE_EVENT_CODES` array rather than just the first index?